src/vs/workbench/services/positronConnections/browser/positronConnectionsService.ts

```diff
@@ -169,10 +145,15 @@ class PositronConnectionsService extends Disposable implements IPositronConnecti
 			this.connections.push(instance);
 		}
 
+		instance.onDidChangeStatus(() => {
```

Comment:
This event handler must be registered.

Reply:
Resolved in fe6a0d009ddab8ff0dc5ac6b3cfd3145ec8cdd97